pulumitest/pulumiTest.go

```diff
@@ -39,6 +43,39 @@ func NewPulumiTest(t PT, source string, opts ...opttest.Option) *PulumiTest {
 	return pt
 }
 
+// NewPulumiYamlTest creates a new PulumiYamlTest instance.
+func NewInlinePulumiTest(t PT, program *pulumiyaml.Program, opts ...opttest.Option) *PulumiTest {
```

Comment:
At minimum, the comment needs to be updated to reflect the renamed function.
```suggestion
// NewInlinePulumiTest creates a PulumiTest instance set up to ...
func NewInlinePulumiTest(t PT, program *pulumiyaml.Program, opts ...opttest.Option) *PulumiTest {
```

Ideally, the comment would be expanded to explain how `NewInlinePulumiTest` should be used, ideally with an example. This is a library function, so most developers will look at the doc comment in their IDE, and it should be useful.